test/e2e/middleware-rewrites/app/next.config.js

```diff
@@ -3,4 +3,16 @@ module.exports = {
     locales: ['ja', 'en', 'fr', 'es'],
     defaultLocale: 'en',
   },
+  rewrites() {
+    return {
+      beforeFiles: [
+        {
+          source: '/config-rewrite-1',
+          destination: '/ab-test/a',
+        },
+      ],
+      afterFiles: [],
```

Comment:
Should we also add one verifying `afterFiles` as well just in case?

Reply:
Yeah adding one